roles/docker_node/tasks/main.yml

```diff
@@ -1,3 +1,20 @@
+# Version unlocking the packages, if you want to update the docker packages.
+- name: install versionlock package
+  package:
+    name: "yum-plugin-versionlock"
+    state: present
```

Comment:
Seems like this should have `when: ansible_os_family == "RedHat"` - same for several tasks below it. You might want to move them into a block as well so that you don't have to repeat the conditional.